lib-java/README.md

```diff
@@ -7,7 +7,7 @@
 
 ### DESCRIPTION
 
-This <a target="_blank" href="https://github.com/signalfx/integrations/tree/release/lib-java">repository</a> contains libraries for instrumenting Java applications and reporting metrics to SignalFx, using Codahale Metrics.
+This <a target="_blank" href="https://github.com/signalfx/signalfx-java">repository</a> contains libraries for instrumenting Java applications and reporting metrics to SignalFx, using Codahale Metrics.
 
 You can also use the module `signalfx-java` to send metrics directly to SignalFx using protocol buffers, without using Codahale or Yammer metrics.
```

Comment:
Same thing here and anywhere else in this file. Change "SignalFx" to "Splunk Infrastructure Monitoring".